ocaml/xenopsd/xc/xenops_server_xen.ml

```diff
@@ -2313,10 +2313,12 @@ module VM = struct
         )
         (create_device_model_config vm vmextra vbds vifs vgpus vusbs) ;
       match vm.Vm.ty with
-      | Vm.PV {vncterm= true; vncterm_ip= ip; _}
-      | Vm.PVinPVH {vncterm= true; vncterm_ip= ip; _} ->
-          Service.PV_Vnc.start ~xs ?ip di.Xenctrl.domid
-      | _ ->
+      | PV {vncterm; vncterm_ip= ip; _}
+      | PVH {vncterm; vncterm_ip= ip; _}
+      | PVinPVH {vncterm; vncterm_ip= ip; _} ->
```

Comment:
`when vncterm ->`

Reply:
That would mean I have to add explicit matches for when vncterm is false, which is what I'm trying to avoid.